src/test/scala-2.12/org/apache/james/gatling/simulation/Configuration.scala

```diff
@@ -7,8 +7,10 @@ import scala.concurrent.duration._
 object Configuration {
 
   val ServerHostName = "127.0.0.1"
-  val BaseJmapUrl = s"http://$ServerHostName:1080"
-  val BaseJamesWebAdministrationUrl = new URL(s"http://$ServerHostName:8000")
+
+  def BaseJmapUrl(getMappedPort: Int => Int) = s"http://$ServerHostName:${getMappedPort(80)}"
```

Comment:
we probably don't need this getMappedPort function

Reply:
Indeed, i thought i needed when i was trying to use test simulation into it simulation...